docs/userguide/containers/dockervolumes.md

```diff
@@ -159,6 +159,44 @@ user with access to host and its mounted directory.
 >should be portable. A host directory wouldn't be available on all potential
 >hosts.
 
+### Mount a shared-storage volume as a data volume
+
+In addition to mounting a host directory to your container, Docker
+enables [volume plugins](../../extend/plugins_volume.md) which allow
+provisioning and mounting of an iSCSI, NFS, or FC volumes. The difference
+with volume plugins is that the data volume is automatically provisioned
+or made available from externally shared storage and mounted to the 
+Docker host where your container runs.
+
+>**Note**: A benefit of using shared volumes is that they are _NOT_ 
+>host-dependent. This means that the volume can be made available to
+>any host that a container is started on as long as it has access to
+> the shared storage backend.
```

Comment:
This should not be a "note", because it's not really an "exceptional" case; this can be regular text.